spec/model/rummager_search_spec.rb

```diff
@@ -3,11 +3,30 @@
 RSpec.describe RummagerSearch, type: :model do
   include ServicesRequestHelpers
 
+  let(:content_id) { "12345" }
+  let(:document_type) { "guide" }
+  let(:search_all_content) { described_class.new(:content_id) }
+  let(:search_document_type) { described_class.new(:content_id, :document_type) }
+
+  describe "#initialize" do
+    it "should be able initialize with only 1 parameter" do
+      expect(search_all_content).to be_instance_of(RummagerSearch)
+      expect(search_all_content.content_id).to eq(:content_id)
+      expect(search_all_content.document_type).to eq(nil)
+    end
+
+    it "should be able initialize with 2 parameter" do
```

Comment:
I'm not sure if these tests are needed. It feels like they are testing that you can pass optional parameters to the class?

Reply:
As discussed, I will remove these tests as it is testing the functionality of the framework, rather than our code. 😄